Merge branch 'master' into tizen_5.5_prepare

journey2w opened the pull request:
### Change Description ###

Merged branch 'master' into tizen_5.5_prepare.

mijin85cho commented:
The file should be checked : 

1. dotnet/getting-started/**app-filtering.md**

   On the master branch, we changed `getting-started` to `get-started`.
   Also, the **app-filtering.md** is moved to under `dotnet/tutorials`, but in this PR we still have one 
more **app-filtering** under `getting-started` folder.

   In conclusion, we have two **app-filtering.md** files.

2. **toc_all.md** 
   We removed other folders information except `Application` on master branch,
   but in this PR other information such as `Design` is still included.

journey2w commented:
> The files should be checked :
> 
> 1. dotnet/getting-started/**app-filtering.md**
>    On the master branch, we changed `getting-started` to `get-started`.
>    Also, the **app-filtering.md** is moved to under `dotnet/tutorials`, but in this PR we still have one
>    more **app-filtering** under `getting-started` folder.
>    In conclusion, we have two **app-filtering.md** files.
> 2. **toc_all.md**
>    We removed other folders information except `Application` on master branch,
>    but in this PR other information such as `Design` is still included.
>    For now we will not work with **toc_all.md** file, but as we will merge it to master branch in the future, it should be handled carefully.

Done.
